posts/GSoC_2019_Project_First_Evaluation.md

```diff
@@ -0,0 +1,74 @@
+---
+title: GSoC 2019 PyCanChemAndYouCan2
+date: 2019-06-22 17:41 UTC-21:41
+slug: GSoC_2019_Project_First_Evaluation
+tags: GSoC 2019
+description: 2019 Google Summer of Code Cnatera Project Blog
+type: text
+author: Chao Xu
+---
+
+# GSoC 2019 PyCanChemAndYouCan2 (ChemCheck) First Evaluation
+
+  Thanks for the instructions from mentors Richard West([@rwest](https://github.com/rwest)), 
+Bryan Weber([@bryanwweber](https://github.com/bryanwweber)) and Kyle Niemeyer([@kyleniemeyer](https://github.com/kyleniemeyer)), the website
+is functioning after first month, and we will keep improving it in the rest of GSoC time. From what I mentioned in last post, ChemCheck is a
+web-based application for visualizing and diagnosing syntax and chemical errors in Chemkin and Cantera input files. In this case, we created basic 
+functions which allow users to convert their mechanism files to Cantera yaml input files easily (Cantera input file is going to be changed
+from cti file to yaml file, so we made some ajustments as well.).
+
+## Upload Page
+
+Users will upload their files to ChemCheck for checking, so we created an upload page which allows users to upload their mechanism, transport,
+thermo, and surface file. Here is the view of upload page.
+
+![image of upload page](https://github.com/12Chao/myproject/blob/master/images/upload%20page.png)
```

Comment:
Please do not put spaces into image filenames. Also, you should put any images into the `images` directory in the repository, in a folder named for this post. Then, you should use the appropriate replacement syntax for the image (its a little different from default Markdown). See here:

https://github.com/Cantera/cantera-website/blob/3d9809107515122b976ab973645395a227a1303d/posts/new-website-is-live.md#L26

and [here](https://getnikola.com/handbook.html#built-in-shortcodes) under the **thumbnail** category.